test/tests/open-liberty-hello-world/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# since we have curl in the liberty image, we'll use that
+clientImage="$1"
+
+serverImage="$1"
+
+# Create an instance of the container-under-test
+cid="$(docker run -d "$serverImage")"
+trap "docker rm -vf $cid > /dev/null" EXIT
+
+_request() {
+	local url="${1#/}"
+	shift
+
+	docker run --rm --link "$cid":open-liberty "$clientImage" \
+		wget -q -O - "$@" "http://open-liberty:9080/$url"
+}
+
+# Make sure that Open Liberty is listening
+. "$dir/../../retry.sh" '_request / &> /dev/null'
+
+# Check that we can request /
+[ -n "$(_request '/')" ]
+
+# Check that the example "Hello World" servlet works
+helloWorld="$(_request '/version.js')"
```

Comment:
This doesn't seem quite right (especially the comment) -- what's this testing?

Reply:
You are right that the comment is wrong. What the test is attempting to do is validate that the docker container has started correctly. When the Open Liberty container starts it will respond to a get request for /version.js. This is capturing the output from the request and  the following line tests that it contains some of the information that is expected to be returned by that file.